### PR TITLE
Fix human readable variants name in case page for SV variants with start chrom != end chrom

### LINKED ISSUE #2950: BND variant involving 2 chroms rendered imprecisely on case page (pinned vars)
This one:

https://scout.scilifelab.se/cust127/GMSB-UP005/sv/variants/ff896f946f06e5a37a0c6cbf80f0d3f8

When it is pinned it's shown as BND(8q24.21-8p11.2) but should be BND(8q24.21-2p11.2).

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #2950 

**How to test**:
1. Look for a BND variant in a case on clinical-db stage
2. Confirm bug described in #2950 
3. Install this branch and make sure that the variant naming is correct

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Tested on clinical-db with this variant: https://scout-stage.scilifelab.se/cust000/F0006358-mip7

Before:

image

After:  🆗 

image

File: scout/server/blueprints/cases/templates/cases/utils.html [hunk 1]
```diff
@@ -312,7 +311,7 @@ <h5 class="modal-title" id="exampleModalLabel">Beacon submission</h5>
                               institute_id=variant.institute,
                               case_name=case.display_name,
                               variant_id=variant._id) }}">
-    {{ variant.sub_category|upper }}({{ variant.chromosome }}{{ variant.cytoband_start }}-{{ variant.chromosome }}{{ variant.cytoband_end }})
+    {{ variant.sub_category|upper }}({{ variant.chromosome }}{{ variant.cytoband_start }}-{{ variant.end_chrom }}{{ variant.cytoband_end }})
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/utils.html:314]:
```suggestion
    {{ variant.sub_category|upper }}({{ variant.chromosome }}{{ variant.cytoband_start }}::{{ variant.end_chrom }}{{ variant.cytoband_end }})
```

Comment (dnil) [reply scout/server/blueprints/cases/templates/cases/utils.html:314]: :: is really ISCN2016, but the new 2020 nomenclature is silly complex using like chr:pos_posdelins[chr:pos_pos] (see eg https://varnomen.hgvs.org/recommendations/DNA/variant/complex/) Either is fine. 😸

**Review by dnil — APPROVED** [review]

File: scout/server/blueprints/cases/templates/cases/utils.html (hunk 1 above)
Comment (northwestwitch) [reply scout/server/blueprints/cases/templates/cases/utils.html:314]: Who am I to challenge ISCN2016? 😄

**Pull request merged**